content/en/docs/getting-started-guides/windows/_index.md

```diff
@@ -3,399 +3,1025 @@ title: Using Windows Server Containers in Kubernetes
+   <a href="https://github.com/openvswitch/ovn-kubernetes">OVN-Kubernetes</a> (uses <a href="http://docs.openvswitch.org/en/latest/intro/install/windows/#open-vswitch-on-windows">OpenVSwitch on Windows</a> - <a href="https://cloudbase.it/openvswitch/#download">Prebuilt certified installer</a>), requires <a href="https://support.microsoft.com/en-us/help/4482887/windows-10-update-kb4482887">KB4482887</a>
+   </td>
+   <td>win-overlay should be used when virtual container networks are desired to be isolated from underlay of hosts (e.g. for security reasons). Allows for IPs to be re-used for different overlay networks (which have different VNID tags)  if you are restricted on IPs in your datacenter. This option may be used when the container hosts are not L2 adjacent but have L3 connectivity 
+   </td>
+  </tr>
+  <tr>
+   <td>Transparent <em>(not used in Kubernetes</em>)
+   </td>
+   <td>Containers are given a vNIC connected to an external vSwitch. Containers are attached to the underlay network directly and the physical network needs to learn the container MACs.
+   </td>
+   <td>Both MAC and IP remains the same.
+   </td>
+   <td>
+   </td>
+   <td>Included here for completeness
+   </td>
+  </tr>
+  <tr>
+   <td>NAT (<em>not used in Kubernetes</em>)
+   </td>
+   <td>Containers are given a vNIC connected to an internal vSwitch. DNS/DHCP is provided using an internal component called <a href="https://blogs.technet.microsoft.com/virtualization/2016/05/25/windows-nat-winnat-capabilities-and-limitations/">WinNAT</a>
+   </td>
+   <td>MAC and IP is rewritten to host MAC/IP.
+   </td>
+   <td><a href="https://github.com/Microsoft/windows-container-networking/tree/master/plugins/nat">nat</a>
+   </td>
+   <td>Included here for completeness
+   </td>
+  </tr>
+</table>
+
+Furthermore, for simplified consumption of the win-bridge and win-overlay plugins for end-users with automated network configurations, [Flannel](https://github.com/coreos/flannel) has added support for Windows in their [meta plugin](https://github.com/containernetworking/plugins/tree/master/plugins/meta/flannel) via the [VXLAN network backend](https://github.com/coreos/flannel/blob/master/Documentation/backends.md#vxlan) (**alpha support** ; delegates to win-overlay) and [host-gateway network backend](https://github.com/coreos/flannel/blob/master/Documentation/backends.md#host-gw) (stable support; delegates to win-bridge) with the [host-local plugin](https://github.com/containernetworking/plugins/tree/master/plugins/ipam/host-local) used for IP address management and the meta CNI network plugin [flannel](https://github.com/containernetworking/plugins/tree/master/plugins/meta/flannel#windows-support-experimental) for interacting between the delegated CNI plugin (win-bridge or win-overlay) and providing the host-local IPAM the correct range from subnet.env file provisioned by the flanneld agent. 
```

Comment:
Can we change this to something like:
"As outlined above, the Flannel CNI meta plugin is also supported on Windows. This plugin supports delegating to one of the reference CNI plugins (win-overlay, win-bridge), to work in conjunction with Flannel daemon on Windows (Flanneld) for automatic node subnet lease assignment and HNS network creation. This plugin reads in its own configuration file (net-conf.json), and aggregates it with the environment variables from the FlannelD generated subnet.env file. It then delegates to one of the reference CNI plugins for network plumbing, and sends the correct configuration containing the node-assigned subnet to the IPAM plugin (e.g. host-local)."

Reply:
made the update almost 90% what you mentioned. i added a couple more links in there as well